config/heracles/quests/appliedenergistics2/Memory Card.json

```diff
@@ -19,6 +19,21 @@
     }
   },
   "rewards": {
+    "dyes": {
```

Comment:
Good choice, adding a dye as reward here!

  BTW... there's a 'selection' type reward, for the player to choose one of many choices as a reward. iirc, the food quest in the _Getting Started_ chapter has three such rewards.

  But for that, one needs to edit the .json file directly. Which can be its own annoying pain: if there's a mistake in formatting the json--heracles may even remove that section of .json, and re-write the whole thing.

  Just so you know, in case you want delve deeper into how Heracles works; it's not necessary to do so.